src/core_plugins/console/public/src/controllers/sense_controller.js

```diff
@@ -16,14 +17,14 @@ module.run(function (Private, $rootScope) {
   };
 });
 
-module.controller('SenseController', function SenseController($scope, $timeout, $location, docTitle) {
-
+module.controller('SenseController', function SenseController(Private, $scope, $timeout, $location, docTitle) {
   docTitle.change('Console');
 
-  let input, output;
+  $scope.topNavController = Private(SenseTopNavController)
 
   // We need to wait for these elements to be rendered before we can select them with jQuery
   // and then initialize this app
+  let input, output;
```

Comment:
ಠ_ಠ

I guess we can let this one slide since it's just copied and pasted from a few lines above...